fpga_interchange/arch.h

```diff
@@ -818,12 +844,39 @@ struct Arch : ArchAPI<ArchRanges>
     void map_cell_pins(CellInfo *cell, int32_t mapping) const;
     void map_port_pins(BelId bel, CellInfo *cell) const;
 
+    TileStatus &get_tile_status(int32_t tile)
+    {
+        auto result = tileStatus.emplace(tile, TileStatus());
+        if (result.second) {
+            auto &tile_type = chip_info->tile_types[chip_info->tiles[tile].type];
+            result.first->second.boundcells.resize(tile_type.bel_data.size());
+            result.first->second.tags.resize(default_tags.size());
+
+            result.first->second.sites.reserve(tile_type.number_sites);
+            for (size_t i = 0; i < tile_type.number_sites; ++i) {
```

Comment:
this is giving a mixed signed/unsigned warning

Reply:
Fixed.